tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -66,7 +66,7 @@ public DashboardView()
         _dispatcher = Microsoft.UI.Dispatching.DispatcherQueue.GetForCurrentThread();
         _localSettingsService = Application.Current.GetService<ILocalSettingsService>();
 
-        ActualThemeChanged += OnActualThemeChanged;
+        Application.Current.GetService<IAdaptiveCardRenderingService>().RendererUpdated += HandleRendererUpdated;

```

Comment:
Consider subscribing and unsubscribing to event from `[RelayCommand] Loaded` and `[RelayCommand] Unloaded` events. Since the service has a longer lifetime (singleton) than the view, the latter (subscriber) will likely be trapped in memory (referenced by the publisher) and won't be garbage collected.

[Unlocking the Secrets of Managed Memory: Dive into Event Handler Leak Insights!](https://devblogs.microsoft.com/visualstudio/unlocking-the-secrets-of-managed-memory-dive-into-event-handler-leak-insights/)

Reply:
Example in [PackageCatalogListViewModel.cs](https://github.com/microsoft/devhome/blob/1fbd2c1375846b949dd3cc03b2553b8b8efa1f64/tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageCatalogListViewModel.cs#L166)